packages/webapp/src/routes/__tests__/land/legal-party-list.spec.js

```diff
@@ -0,0 +1,93 @@
+import { submitGetRequest } from '../helpers/server.js'
+import constants from '../../../utils/constants.js'
+const url = constants.routes.LEGAL_PARTY_LIST
+
+describe('url', () => {
+  let viewResult
+  let h
+  let redisMap
+  let resultContext
+  let legalPartyList
+
+  beforeEach(() => {
+    h = {
+      view: (view, context) => {
+        viewResult = view
+        resultContext = context
+      },
+      redirect: (view, context) => {
+        viewResult = view
+      }
+    }
+
+    redisMap = new Map()
+    redisMap.set(constants.redisKeys.LEGAL_AGREEMENT_PARTIES, [
+      {
+        organisationName: 'org1',
+        organisationRole: 'Developer',
+        organisationOtherRole: 'undefined'
+      },
+      {
+        organisationName: 'org2',
+        organisationRole: 'Landowner',
+        organisationOtherRole: 'undefined'
+      }
+    ])
+
+    legalPartyList = require('../../land/legal-party-list.js')
+  })
+
+  describe('GET', () => {
+    it(`should render the ${url.substring(1)} view`, async () => {
+      const response = await submitGetRequest({ url })
+      expect(response.statusCode).toBe(200)
```

Comment:
By default submitGetRequest will already assert a 200 response, unless a different http code is passed through